README.md

```diff
@@ -39,24 +39,28 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 <a name="prerequisites"></a>
 ### Prerequisites
 
-Minishift requires a hypervisor to run the virtual machine containing OpenShift. Depending on your
-host OS, you have the choice of the following hypervisors:
+Minishift requires a hypervisor to run the virtual machine containing OpenShift.
```

Comment:
instead of `run the VM containing OpenShift`, good to have `start the VM for OpenShift single node cluster`.